README.md

```diff
@@ -1,10 +1,212 @@
+
+Install the SDK and the ethers package. _**Note**_ Aragon SDK uses ethers
+version5. Although version 6 may work, we suggest using the same version5 for
+compatibility
+
+```sh
+npm install @aragon/sdk-client @ethers@5
+```
+
+optionally you can also enable typescript
+
+```sh
+npx tsc --init
+```
+
+even if you are building with it will give you access to better intellisense
+
+<br/>
+
+### 3. Create an entry File
+
+create a new file `index.ts` (or `index.js`) and import ethers and the SDK. This
+is where we will setup the SDK and call its functions
+
+```typescript
+import { Wallet } from "ethers";
+import { Client, Context, ContextParams } from "@aragon/sdk-client";
+
+const main = async () => {
+  // ...
+};
+
+main()
+  .then(() => process.exit(0))
+  .catch((error) => {
+    console.log(error);
+    process.exit(1);
+  });
+```
+
+#### 4. Configure the SDK
+
+In order to set up the configuration for the Aragon SDK, we want to create a
+"context" variable which will give us access to the web3 provider and signer
+handling the SDK transactions.
+
+```typescript
+const contextParams: ContextParams = {
+  network: "mainnet",
+  web3Providers: "https://rpc.ankr.com/eth",
+  signer: Wallet.createRandom(),
+  // Optional, but without it the client will not be able to resolve IPFS content
+  ipfsNodes: [
+    {
+      url: "https://testing-ipfs-0.aragon.network/api/v0",
+      headers: { "X-API-KEY": "b477RhECf8s8sdM7XrkLBs2wHc4kCMwpbcFC55Kt" },
+    },
+  ],
+};
+
+const context: Context = new Context(contextParams);
+```
+
+Next thing you'll want to do is set up the general purpose client so you can
+call on the SDK functions. This client is used to interact with any DAO on the
+network you're connected to.
+
+```typescript
+const client: Client = new Client(context);
+```
+
+<br/>
+
+### 5. Using the SDK
+
+Now test it's all working by fetching information about the Aragon Association
+multisig
+
+```typescript
+console.log(await client.methods.getDao("aa.dao.eth"));
+```
+
+our final code should look like this
+
+```typescript
+import { Wallet } from "ethers";
+import { Client, Context } from "@aragon/sdk-client";
+
+const main = async () => {
+  const contextParams: ContextParams = {
+    network: "mainnet",
+    web3Providers: "https://rpc.ankr.com/eth",
+    signer: Wallet.createRandom(),
+    // Optional, but without it the client will not be able to resolve IPFS content
+    ipfsNodes: [
+      {
+        url: "https://testing-ipfs-0.aragon.network/api/v0",
+        headers: { "X-API-KEY": "b477RhECf8s8sdM7XrkLBs2wHc4kCMwpbcFC55Kt" },
+      },
+    ],
+  };
+
+  const context: Context = new Context(contextParams);
+  const client: Client = new Client(context);
+
+  console.log(await client.methods.getDao("aa.dao.eth"));
+};
+
+main()
+  .then(() => process.exit(0))
+  .catch((error) => {
+    console.log(error);
+    process.exit(1);
+  });
+```
+
+Now run the program, in out console we should see the information related to the
```

Comment:
"In order to run the program and get the information details on the `aa.dao.eth` DAO, go to the terminal and run 
```
node index.ts
```
"

No need to specify it's the Aragon Association multisig DAO.